moto/ec2/models.py

```diff
@@ -5903,13 +5904,26 @@ def delete_customer_gateway(self, customer_gateway_id):
 
 class TransitGateway(TaggedEC2Resource, CloudFormationModel):
 
+    DEFAULT_OPTIONS = {
+        "AmazonSideAsn": "64512",
+        "AssociationDefaultRouteTableId": "tgw-rtb-0d571391e50cf8514",
+        "AutoAcceptSharedAttachments": "disable",
+        "DefaultRouteTableAssociation": "enable",
+        "DefaultRouteTablePropagation": "enable",
+        "DnsSupport": "enable",
+        "MulticastSupport": "disable",
+        "PropagationDefaultRouteTableId": "enable",
+        "TransitGatewayCidrBlocks": None,
+        "VpnEcmpSupport": "enable"
+    }
+
     def __init__(self, backend, description=None, options=None, tags=[]):
         self.ec2_backend = backend
         self.id = random_transit_gateway_id()
         self.description = description
         self.state = "available"
         self.add_tags(tags or {})
-        self.options = options
+        self.options = patch_dict(self.DEFAULT_OPTIONS, options)
```

Comment:
I think you want to reverse the order of these two arguments (to leave `DEFAULT_OPTIONS` unchanged):
```
         self.options = patch_dict(options or {}, self.DEFAULT_OPTIONS)
```

Reply:
Created new method